src/visitors/rules/structural/validateClientCredentialsOpenAPIFlow.js

```diff
@@ -7,29 +7,33 @@ class ValidateClientCredentialsOpenAPIFlow {
     return {
       tokenUrl(node, ctx) {
         if (!node.tokenUrl) return ctx.createError(ctx.messageHelpers.missingRequiredField('tokenUrl'), 'key');
-        if (typeof node.tokenUrl !== 'string') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        if (typeof node.tokenUrl !== 'string') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        }
         return null;
       },
       refreshUrl(node, ctx) {
-        if (node.refreshUrl && typeof node.refreshUrl !== 'string') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        if (node.refreshUrl && typeof node.refreshUrl !== 'string') {
+          return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('string'), 'value');
+        }
         return null;
       },
       scopes(node, ctx) {
         const wrongFormatMap = Object.keys(node.scopes)
           .filter((scope) => typeof scope !== 'string' || typeof node.scopes[scope] !== 'string')
           .length > 0;
-        if (wrongFormatMap) return ctx.createError('The scopes field must be a Map[string, string] in the OpenAPI Flow Object', 'value');
+        if (wrongFormatMap) {
+          return ctx.createError('The scopes field must be a Map[string, string] in the OpenAPI Flow Object', 'value');
```

Comment:
I think this is misnamed. I believe they refer to it as the [OAuth flows object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#oauthFlowsObject).... Also, check the previous file too.